staging/src/k8s.io/apiextensions-apiserver/go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/google/gofuzz v1.1.0
 	github.com/google/uuid v1.1.1
 	github.com/googleapis/gnostic v0.4.1
+	github.com/pkg/errors v0.9.1
```

Comment:
I'm not sure if it's okay to introduce this dependency here, which is why i didn't update vendors yet to keep this readable.
I would prefer to use the new wrap mechanics of go1.13 but then the change couldn't be backported. Would that be possible anyways?